spec/features/select_2_spec.rb

```diff
@@ -389,5 +389,27 @@ def create_items
         expect(page).to have_css(".select2-container .select2-chosen", text: /My shiny Cat #2/)
       end
     end
+
+    context "with different order" do
+      before do
+        register_page(Category) {}
+
+        register_form(Invoice, false) do |f|
+          f.input :category_id,
+                  as: :search_select,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.